ecosystem/sep-0008.md

```diff
@@ -61,107 +72,211 @@ approval_server="https://goat.io/tx_approve"
 approval_criteria="The goat approval server will ensure that transactions are compliant with NFO regulation"
 ```
 
+## Transaction Composition
+
+A transaction that an approval server will approve consists of operations that authorize the holding account, transact the regulated asset, and deauthorize the holding account. If a transactions is built this way, there will be no point where holding accounts have open, authorized trustlines to run unapproved operations because of transaction atomocity.
+
+A sandwich transaction described above can be built by wallets preemptively or by approval servers in order to make it compliant when the [revised](#revised) status is applicable.
```

Comment:
💡 As it occurs to me the term sandwich in this sentence doesn't make it easier to understand what's happening, and so I think unnecessarily introduces a new concept to the vocabulary of this SEP. Could you leave it out?
```suggestion
A transaction as described above can be built by wallets preemptively or by approval servers in order to make it compliant when the [revised](#revised) status is applicable.
```

Reply:
https://github.com/stellar/stellar-protocol/pull/817/files#diff-dfd6de7cb4877cfd8a07c699a07a96d254ba8fa32c3d7af268393901f3a08563R76